store/tikv/txn.go

```diff
@@ -369,6 +369,15 @@ func (txn *tikvTxn) rollbackPessimisticLocks() error {
 // lockWaitTime in ms, except that kv.LockAlwaysWait(0) means always wait lock, kv.LockNowait(-1) means nowait lock
 func (txn *tikvTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keysInput ...kv.Key) error {
 	// Exclude keys that are already locked.
+	var err error
+	defer func() {
+		if err == nil {
+			if lockCtx.PessimisticLockWaited > 0 {
+				lockCtx.LockTimeWaited = time.Since(lockCtx.WaitStartTime)
```

Comment:
Seems the field `LockTimeWaited` is unnecessary.

Reply:
This is used for slow query logging in the diff part2